aws/sdk-codegen/src/main/resources/default-partitions.json

```diff
@@ -0,0 +1,105 @@
+{
```

Comment:
How is this file kept up to date?

Reply:
yeah we'll need to sync it the same way we sync endpoints.json